core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -27,12 +27,13 @@ import java.util.concurrent.atomic.{AtomicReference, AtomicBoolean, AtomicIntege
 import java.util.UUID.randomUUID
 
 import scala.collection.{Map, Set}
-import scala.collection.JavaConversions._
 import scala.collection.generic.Growable
+import scala.collection.JavaConversions._
 import scala.collection.mutable.HashMap
 import scala.reflect.{ClassTag, classTag}
 import scala.util.control.NonFatal
 
+import com.google.common.annotations.VisibleForTesting
```

Comment:
I don't mind it, but I haven't seen this annotation being used in Scala code inside Spark.